example-app/app/Http/Requests/Api/Movie/CreateMovie.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+namespace App\Http\Requests\Movie;
```

Comment:
папка и namespace отличается
это нарушение стандарта
```suggestion
namespace App\Http\Requests\Api\Movie;
```
перепроверь остальные requests